src/components/GrowlNotification/index.tsx

```diff
@@ -16,15 +19,27 @@ const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
 
-function GrowlNotification(_, ref) {
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Is this because `_`? I'm not completely sure if the linter isn't in the right here. Do we commonly use this convention?

Reply:
Yes, naming convention in eslint is setup so we can't use arguments like `_`, `_props` etc. Here props aren't used so `_` name is fine with me.

Reply:
**Our** ESLint is configured so we can't use arguments starting with `_`. So, either we accepted the defaults or explicitly decided to use this.

While I agree with you on some level, and I know the `_` naming convention is general, it seems that Expensify explicitly opted out of using it.

We should either remove this linter config or change the name.

Reply:
> While I agree with you on some level, and I know the _ naming convention is general, it seems that Expensify explicitly opted out of using it.

I configured this ESlint rule, and forgot about arguments starting with `_` (I configured it when there were no TS files in the codebase)

> We should either remove this linter config or change the name.

We already use arguments starting with `_` in multiple places. How about adjusting the ESlint rule so it allows arguments starting with `_`?

Reply:
> How about adjusting the ESlint rule so it allows arguments starting with _?

Fine for me! I'm just ensuring that we don't fight with our own configuration for no good reason.

Reply:
@cubuspl42 @blazejkustra I've updated the rule to allow the usage of leading underscores and cleaned up the eslint-disables that were used as a workaround.